common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1463,21 +1487,42 @@ object CelebornConf extends Logging {
       .checkValue(Seq("roundrobin", "loadaware").contains(_), "")
       .createWithDefault("roundrobin")
 
-  def initialPartitionSize(conf: CelebornConf): Long = {
-    Utils.byteStringAsBytes(conf.get("rss.initial.partition.size", "64m"))
-  }
 
-  def minimumPartitionSizeForEstimation(conf: CelebornConf): Long = {
-    Utils.byteStringAsBytes(conf.get("rss.minimum.estimate.partition.size", "8m"))
-  }
+  val INITIAL_ESTIMATED_PARTITION_SIZE: ConfigEntry[Long] =
+    buildConf("celeborn.worker.initialEstimatedPartitionSize")
+      .withAlternative("rss.initial.partition.size")
+      .categories("master")
+      .doc("Initial estimated partition size, default is 64m, and it will change according to runtime stats.")
+      .version("0.2.0")
+      .bytesConf(ByteUnit.BYTE)
+      .createWithDefaultString("64mb")
 
-  def partitionSizeUpdaterInitialDelay(conf: CelebornConf): Long = {
-    Utils.timeStringAsMs(conf.get("rss.partition.size.update.initial.delay", "5m"))
-  }
+  val MINIMUM_ESTIMATED_PARTITION_SIZE: ConfigEntry[Long] =
+    buildConf("celeborn.worker.minimumEstimatedPartitionSize")
+      .withAlternative("rss.minimum.estimate.partition.size")
+      .categories("master")
+      .doc("Ignore partition size smaller than this configuration for partition size estimation.")
+      .version("0.2.0")
+      .bytesConf(ByteUnit.BYTE)
+      .createWithDefaultString("8mb")
 
-  def partitionSizeUpdateInterval(conf: CelebornConf): Long = {
-    Utils.timeStringAsMs(conf.get("rss.partition.size.update.interval", "10m"))
-  }
+  val ESTIMATED_PARTITION_SIZE_UPDATE_INITIAL_DELAY: ConfigEntry[Long] =
+    buildConf("celeborn.worker.estimatedPartitionSize.update.initialDelay")
+      .withAlternative("rss.partition.size.update.initial.delay")
+      .categories("master")
+      .doc("Initial delay time before start updating estimated partition size.")
+      .version("0.2.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("5m")
```

Comment:
better to use 5min, to distinguish w/ megabytes

Reply:
DOne